src/solver/conv_hip_implicit_gemm_mlir_cpp_fwd.cpp

```diff
@@ -143,7 +142,9 @@ ConvSolution ConvHipImplicitGemmMlirCppFwd::GetSolution(const ConvolutionContext
 
     size_t local_size  = 0;
     size_t global_size = 0;
+#if MIOPEN_USE_MLIR
```

Comment:
I recommend wrapping the whole function body in this #if.

[Reminder] `std:ignore = <unused param>` under #else to avoid tidy warnings.

Reply:
Both `local_size` and `global_size` are actually used a couple of lines below. They are pushed to the `l_wk`, `g_wk` vectors. 

The thing with commenting out the body is that I still have to keep the `result` local variable and return statement such that it doesn't complain. I can't tell which is cleaner.

Reply:
`return {};`.

This is cleaner -- the reader would not need to think why the half-cooked result is returned.